tests/test_or_optimizer.py

```diff
@@ -0,0 +1,43 @@
+from snuba_sdk.column import Column
```

Comment:
Can you also add a test that makes sure that this optimization gets run for MQL queries and not SnQL queries? Basically something that asserts that the optimizer gets called when it's supposed to

Reply:
I added end-to-end tests for MQL and SnQL serialize